README.rst

```diff
@@ -168,22 +200,22 @@ automatically like you can do with the admin autodiscover. Do so by running
 More
 ====
 
-There is so much more to explore which might be useful for you and your
+There is so much more to explore which might be useful tofrom  you and your
 projects:
 
 * There are ways to register custom ``AutoFixture`` subclasses with models
   that are automatically used when calling ``loadtestdata`` on the model.
 * More control for related models, even with relations of related models...
   (e.g. by using ``generate_fk=['author', 'author__user']``)
-* Custom constraints that are used to ensure that created the models are
-  valid (e.g. ``unique`` and ``unique_together`` constraints which are
+* Custom constraints that are used to ensure that created models are
+  valid (e.g. ``unique`` and ``unique_together`` constraints, which are
   already handled by default)
 
-I hope to explain this in the future with more details in a documentation. It
-will be written but is not finished yet. I wanted to get this project out to
-support you in development. But since its only python code you can easily
-study the source on your own and see in which ways it can be used. There are
-already some parts documented with doc strings which might also be helpful for you.
+I hope to explain this in the future with more detailed documentation. I am in the 
+process of writing it, but I am not finished.  I wanted to get this project out to
+support your development. But, since it's only python code you can easily change it to 
+suit your needs. There are already some parts documented with doc strings which 
+might also be helpful for you.
```

Comment:
How do you feel about this passage? I actually feel guilty since I lied that the docs will come … I haven't gotten around to it in the last six years. What do you think? Shall we remove it?